<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>718</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Comptabilité</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="Title">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="Date">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="NameLabel">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="AdressLabel">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="PostalCodeAndTownLabel">
       <property name="text">
        <string>Code postal et ville</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="NameTextEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="AdressTextEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="PostalCodeAndTownTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="2" column="1">
      <widget class="QTextEdit" name="AdressTextEdit_2">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3 rue Saint Michel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="PostalCodeAndTownTextEdit_2">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;01130 NANTUA &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="NameTextEdit_2">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Nova Bijoux Création&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>26</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>27</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>28</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>29</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>31</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>32</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>33</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>34</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>35</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>36</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>37</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>38</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>39</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>40</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>41</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>42</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>43</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>44</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>45</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>46</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>47</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>48</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>49</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>50</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Designation</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DutyFreePrice</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DutyFreeAmount</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>500</y>
      <width>221</width>
      <height>137</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_DuttyFreePrice">
       <property name="text">
        <string>Prix HT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_TVARatio">
       <property name="text">
        <string>Taux TVA (en %)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_TVAAmount">
       <property name="text">
        <string>Montant TVA</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_TTCPrice">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Prix TTC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="DuttyFreePriceLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="TVARatioLineEdit">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="TVAAmountLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="TTCPriceLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="Line" name="line">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Calculate_pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Save_pushButton">
       <property name="text">
        <string>Sauvegarder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Load_pushButton">
       <property name="text">
        <string>Charger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Print_pushButton">
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Mail_pushButton">
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
